src/cargo/ops/vendor.rs

```diff
@@ -618,10 +618,6 @@ fn vendor_this(relative: &Path) -> bool {
         // Temporary Cargo files
         Some(".cargo-ok") => false,
```

Comment:
Now that we are doing fresh extractions, is this still needed?

Reply:
For git checkouts we still have that, though there is no test exercising that code path. Do you want a test for it in this PR?

Reply:
Let's go ahead and merge this